pkg/apis/serving/v1alpha1/kfservice_framework_scikit.go

```diff
@@ -17,17 +17,41 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
-func (s *ScikitLearnSpec) CreateModelServingContainer(modelName string) *v1.Container {
-	// TODO: https://github.com/kubeflow/kfserving/issues/58
+const (
+	SKLearnEntrypointCommand = "python"
+	SKLearnServingGRPCPort   = "9000"
+	SKLearnServingRestPort   = "8080"
+	SKLearnServingImageName  = "animeshsingh/sklearnserver"
```

Comment:
Yeah we are good here - not passing them down the line for instantiation. it would be good to allow in future, in case someone needs to change.

// "--port=" + SKLearnServingGRPCPort,
// "--rest_api_port=" + SKLearnServingRestPort